lib/cassia/access_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Cassia
+  class AccessController
+    include Virtus.model
+
+    attribute :access_token, String, default: nil
```

Comment:
do we have to default these to `nil`? If they are defaulted to `nil` I think we don't need to specify the defaults